### PR TITLE
pass telemetry config to docfx

### PR DESCRIPTION
**Pull request by 928PJY**
implement: https://dev.azure.com/ceapex/Engineering/_workitems/edit/171978

### CODE REVIEW

File: src/build/buildExecutor.ts [hunk 1]
```diff
@@ -38,15 +46,19 @@ export class BuildExecutor {
         let outputPath = path.join(input.localRepositoryPath, OUTPUT_FOLDER_NAME);
         fs.emptyDirSync(outputPath);
 
-        let envs = {
+        let envs: any = {
+            'DOCFX_CORRELATION_ID': correlationId,
```
Comment (yufeih) [inline src/build/buildExecutor.ts:50]: Do you need changes in docfx to support more common properties? I recall there is a need to tell if docfx is running local build or not

Comment (928PJY) [reply src/build/buildExecutor.ts:50]: Yeah, you are right, I will update spec too. how about calling it `IsLocalBuild`? and use the command line parameter `--local-build`?

**Review by live1206 — APPROVED** [review]

File: src/build/buildExecutor.ts (hunk 1 above)
Comment (yufeih) [reply src/build/buildExecutor.ts:50]: In the originally design, the telemetry is a dictionary that allows you to add custom dimensions to docfx. Let me change docfx first, but you are good with this change.

Comment (928PJY) [reply src/build/buildExecutor.ts:50]: Cool, I will add a TODO on the extension side

**Pull request merged**